db/migrate/20151435234635_add_container_definition_security_contexts.rb

```diff
@@ -0,0 +1,13 @@
+class AddContainerDefinitionSecurityContexts < ActiveRecord::Migration
+  def change
+    add_column    :container_definitions, :privileged, :boolean
+    add_column    :container_definitions, :run_as_user, :bigint
+    add_column    :container_definitions, :se_linux_user, :string
+    add_column    :container_definitions, :se_linux_role, :string
+    add_column    :container_definitions, :se_linux_type, :string
+    add_column    :container_definitions, :se_linux_level, :string
```

Comment:
I did something similar in the beginning but have been told to change it that way:  https://github.com/ManageIQ/manageiq/pull/4010#discussion_r37754743 cc @simon3z